src/style/foundation/video-calling.less

```diff
@@ -180,7 +163,7 @@
     cursor: pointer;
     line-height: 0;
 
-    @media (max-height: @screen-height-sm) {
+    @media (max-width: @call-ui-xs) {
       margin: 0 5px;
     }
```

Comment:
The vertical media-query wasn't doing much, but reducing the margin for the smallest width breakdown allow to reach the target 320px